WordPress/src/main/java/org/wordpress/android/viewmodel/pages/PagesViewModel.kt

```diff
@@ -475,6 +476,14 @@ class PagesViewModel
 
     private fun publishPageNow(remoteId: Long) {
         _publishAction.value = pageMap[remoteId]
+        launch(uiDispatcher) {
+            withContext(defaultDispatcher) {
```

Comment:
I think we can remove `withContext(defaultDispatcher)` as delay takes care of not blocking the current thread. Wdyt?

Reply:
Sure! Sounds like a good idea!